bitnami/contour/templates/contour/deployment.yaml

```diff
@@ -107,7 +107,11 @@ spec:
             - --contour-cafile=/certs/ca.crt
             - --contour-cert-file=/certs/tls.crt
             - --contour-key-file=/certs/tls.key
+            {{- if .Values.contour.configPath }}
             - --config-path=/config/contour.yaml
+            {{- else }}
+            - --contour-config-name= {{ .Values.contour.contourConfigName }}
```

Comment:
Just notice this extra space that I think you should remove

```suggestion
            - --contour-config-name={{ .Values.contour.contourConfigName }}
```

Reply:
I will do it